fs/accounting/accounting.go

```diff
@@ -256,11 +256,14 @@ func (acc *Account) String() string {
 			etas = "0s"
 		}
 	}
+
 	name := []rune(acc.name)
 	if fs.Config.StatsFileNameLength > 0 {
 		if len(name) > fs.Config.StatsFileNameLength {
-			where := len(name) - fs.Config.StatsFileNameLength
-			name = append([]rune{'.', '.', '.'}, name[where:]...)
+			prefixLength := (fs.Config.StatsFileNameLength + 1) / 2
```

Comment:
I think you'd be better off writing

```
suffixLength := fs.Config.StatsFileNameLength / 2
prefixLength := fs.Config.StatsFileNameLength - suffixLength 
```

That is clearer to me that `suffixLength + prefixLength ==  fs.Config.StatsFileNameLength` which is the invariant we are looking for I think.

I think what you wrote does the same thing, I just have more trouble convincing myself of it ;-)

Reply:
As far as i know it does :) 
(if the length is odd lets say 7, then 7/2 = 3 and 8/2 = 4, so it adds up to 7, and when it is even like 6 is, then 6/2 = 3 and 7/2 =3 so it adds up to 6)
But I agree it would look much simpler your way :)